release-notes/1.19.0.md

```diff
@@ -0,0 +1,84 @@
+* You can now use the **Sharing** button on the session page to invite users without an account to join the stream using a link.
+
+#### Applications
+* An application detail page is added where you can see the details of the application and its versions. Click on the application name in the list of applications to view the application detail page.
+* You can manage application versions through the dashboard now. On the **Versions** tab of the application detail page, there's an **Actions** button that allows you to manage versions of the application.
+
+#### Containers
+* You can perform bulk actions on containers on the containers list page. Select one or more containers for the **Start**/**Stop** buttons to appear on top.
+* The container list **Search and filter** feature is enhanced for easier filtering based on status, type of containers, running applications, and the node on which it is running.
+
+### Other
+
+* Android security updates for August 2023 (see [Android Security Bulletin - August 2023](https://source.android.com/docs/security/bulletin/2023-08-01) for more information).
+* The Android WebView has been updated to [115.0.5790.138](https://chromereleases.googleblog.com/2023/07/chrome-for-android-update_25.html).
+* The Mesa driver/VirGL renderer have been updated to the latest upstream version 23.1.<!--AC-1614-->
+* License information for used software components is available in the `NOTICE` file in the root file system.<!--AC-1663-->
+* A verification for secure boot and machine owner key is added to avoid appliance installation failures.<!--AC-1692-->
+* The `wait-for-if.sh` script in the LXD charm now uses `ip` instead of `ifconfig`.<!--AC-1696-->
+* Improved `anbox-cloud-tests` so that it does not delete containers/sessions other than those that it owns.<!--AC-1639-->
+
+## Removed functionality
+
+* Ubuntu 18.04 (bionic) images that were deprecated in 1.16.0 are no longer supported and are removed.
+* The `qemu-props` binary from the Android image was dropped as it is unnecessary and not energy efficient if the service is unused yet running during Android runtime.<!--AC-1599-->
+* The outdated `ashmem` module was originally removed in the 1.17.2 release but AMS still attached the `ashmem` device by default when starting a container. This could cause start up failures and hence with 1.19.0, `ashmem` module is completely removed from AMS and Anbox Cloud.<!--AC-1531-->
+* The following services that consume unnecessary CPU time when launching a container are masked: <!--AC-1793-->
+```
+e2scrub_reap.service
+systemd-journald.service
+systemd-tmpfiles-setup-dev.service
+systemd-journal-flush.service
+systemd-tmpfiles-setup.service
+```
+
+## Deprecations
+
+* Android 11 images are deprecated. See [provided images](https://anbox-cloud.io/docs/ref/provided-images) for a list of supported and deprecated images.
+* Support for Ubuntu 20.04 (focal) hosts is deprecated.
```

Comment:
Should we not say when we will remove these? For Android 11 we will drop in 1.21 in Feb next year and Ubuntu 20.04 I think we said we're going to drop around the same time. @ajanon what have we discussed here?

Reply:
For Android 11, we scheduled to remove it for 1.21, assuming that Google will stop providing security patches around February next year.
For Focal, we wanted to remove it around the time 24.04 becomes available. 1.21 might be a bit early, though. What do you think?

Reply:
With other dependencies and assumptions in future, is it a good idea to call out when we will remove it? 
If you still wish to, it is fine to mention tentative timelines along with the dependencies. We could say something like this:

> * Android 11 images are deprecated since 1.19.0 and are planned to be removed when they no longer get security patches from Google. This is expected to happen around the Anbox Cloud 1.21 release. See [provided images](https://anbox-cloud.io/docs/ref/provided-images) for the current list of supported and deprecated images.
> * Support for Ubuntu 20.04 (focal) hosts is deprecated and will be removed when Ubuntu 24.04 is available.

What do you think @ajanon @morphis ?

Reply:
Sounds good to me!